ateam_ai/src/behavior.hpp

```diff
@@ -0,0 +1,36 @@
+struct Behavior {
+  enum Type {
+    GetBall,
+
+    // Straight kick
+    MovingKick,
+    PivotKick,
+
+    // Receive + kick
+    OneTouchReceiveKick,
+
+    // Kick + instantaneous target selection near kick time
+    Shot,
+    OneTouchShot,
+
+    // Normal moves
+    MoveToPoint,
+    CostFunctionPoint
+  } type;
+
+  enum Priority {
+    Required,
+    Medium,
+    Low
+  } priority;
```

Comment:
Required is absolutely needed for the behavior sequence, Medium and low priority describe which background behaviors to fill first if there aren't enough robots to fill them all.